servers/republik/modules/crowdfundings/lib/CustomPackages/rules/earlyAdopterBonus.js

```diff
@@ -0,0 +1,57 @@
+const debug = require('debug')('crowdfundings:lib:CustomPackages:rule:earlyUserBonus')
+const moment = require('moment')
+const uuid = require('uuid/v4')
+
+const MEMBERSHIP_CREATED_BEFORE = '2017-04-27'
+
+module.exports = ({ package_, packageOption, membership, payload, now }) => {
+  if (membership.createdAt >= moment(MEMBERSHIP_CREATED_BEFORE)) {
```

Comment:
ideally the rule would be
```
membership.pledge.createdAt && membership.pledge.package.name !== 'ABO_GIVE'
```

currently memberships from `ABO_GIVE` gen on 26. would get it too right?

Reply:
> currently memberships from ABO_GIVE gen on 26. would get it too right?

Yes.

Excluding `ABO_GIVE` memberships should be fairly straight forward.

Reply:
Adopted in 13a5e29